src/utils/cli-utils.test.ts

```diff
@@ -3,7 +3,7 @@ import { SyntheticEvent } from 'react';
 import { copyToClipBoardUtility, getCLISetConfigRegistry } from './cli-utils';
 
 describe('copyToClipBoardUtility', () => {
-  let originalGetSelection;
+  let originalGetSelection: typeof window.getSelection;
```

Comment:
Maybe also a good idea to use `jest.spyOn` to test the clipboard in the future so it's not needed to reset `window.getSelection` manually, like it's done now. But as said the focus for this PR was to do no actual code changes, so I've left it untouched.

Reply:
agree